frontend/src/utils/aboutData.ts

```diff
@@ -4,6 +4,104 @@ export const aboutText = [
   'The code is licensed under the MIT License, encouraging contributions while protecting the authors from legal claims. All OWASP Nest leaders are certified ISC2 professionals and OWASP members who adhere to the OWASP Code of Conduct.',
 ]
 
+// TypeScript interfaces
+export interface KeyFeature {
+  title: string
+  description: string
+}
+
+export interface ProjectHistory {
+  year: string
+  title: string
+  description: string
+}
+
+export interface GetInvolved {
+  description: string
+  ways: string[]
+  callToAction: string
+}
+
+export const missionContent = {
+  mission:
+    'OWASP Nest is a comprehensive platform built to enhance collaboration and streamline contributions across the OWASP community. Acting as a central hub, it helps users discover projects, find contribution opportunities, and connect with like-minded individuals based on their interests and expertise.',
+  whoItsFor:
+    "OWASP Nest is designed for developers, designers, technical writers, students, security professionals, and contributors of all backgrounds. Whether you're just starting out or a seasoned OSS veteran, Nest provides intuitive tools to help you engage meaningfully in the OWASP ecosystem.",
+} as const
+
+export const keyFeatures: KeyFeature[] = [
+  {
+    title: 'Advanced Search Capabilities',
+    description:
+      'Easily filter and explore projects or issues using keywords, tags, and contributor preferences.',
+  },
+  {
+    title: 'Slack Integration',
+    description:
+      'Stay connected through a Slack bot that delivers updates and supports both direct and channel messaging.',
+  },
+  {
+    title: 'OWASP Chapters Proximity Page',
+    description: 'Discover and connect with nearby OWASP chapters for local engagement.',
+  },
+  {
+    title: 'AI-Generated Insights',
+    description:
+      'Benefit from AI-powered summaries and actionable suggestions for tackling project issues.',
+  },
+]
+
+export const getInvolvedContent: GetInvolved = {
+  description:
+    "OWASP Nest thrives thanks to community-driven contributions. Here's how you can make an impact:",
+  ways: [
+    'Code Contributions – Fix bugs or build new features',
+    'Code Review – Improve quality by reviewing pull requests',
+    'Documentation – Create or enhance onboarding guides and tutorials',
+    'Issue Reporting – Report bugs or propose improvements',
+    'Community Engagement – Join Slack discussions and provide feedback',
+  ],
+  callToAction:
```

Comment:
This should be clickable and lead to GitHub repository or other useful landing page.